auth0/src/test/java/com/auth0/android/provider/WebAuthProviderTest.kt

```diff
@@ -2724,6 +2727,299 @@ public class WebAuthProviderTest {
+        assertThat(uri, `is`(notNullValue()))
+        val sentState = uri?.getQueryParameter(KEY_STATE)
+        val sentNonce = uri?.getQueryParameter(KEY_NONCE)
+
+        val intent = createAuthIntent(
+            createHash(
+                null,
+                null,
+                null,
+                null,
+                null,
+                sentState,
+                null,
+                null,
+                "1234"
+            )
+        )
+
+        val jwtBody = JwtTestUtils.createJWTBody()
+        jwtBody["nonce"] = sentNonce
+        jwtBody["aud"] = proxyAccount.clientId
+        jwtBody["iss"] = proxyAccount.getDomainUrl()
+        val expectedIdToken = JwtTestUtils.createTestJWT("RS256", jwtBody)
+        val codeCredentials = Credentials(
+            expectedIdToken,
+            "codeAccess",
+            "DPoP", // Token type should be DPoP when DPoP is enabled
+            "codeRefresh",
+            expiresAt,
+            "codeScope"
+        )
+
+        Mockito.doAnswer {
+            callbackCaptor.firstValue.onSuccess(codeCredentials)
+            null
+        }.`when`(pkce).getToken(eq("1234"), callbackCaptor.capture())
+
+        Assert.assertTrue(resume(intent))
+        mockAPI.takeRequest()
+        ShadowLooper.idleMainLooper()
+        verify(authCallback).onSuccess(credentialsCaptor.capture())
+        val credentials = credentialsCaptor.firstValue
+        assertThat(credentials, `is`(notNullValue()))
+        assertThat(credentials.type, `is`("DPoP"))
+        mockAPI.shutdown()
+    }
+
+    // Update the existing test that checks DPoP key generation
+//    @Test
+//    public fun enablingDPoPWillGenerateNewKeyPairIfOneDoesNotExist() {
+//        `when`(mockKeyStore.hasKeyPair()).thenReturn(false)
+//
+//        WebAuthProvider.useDPoP()
+//            .login(account)
+//            .start(activity, callback)
+//
+//        verify(mockKeyStore).generateKeyPair(any())
+//    }
+//
+//    // Update the existing test for DPoP JWK parameter
+//    @Test
+//    public fun shouldHaveDpopJwkOnLoginIfDPoPIsEnabled() {
+//        `when`(mockKeyStore.hasKeyPair()).thenReturn(true)
+//        `when`(mockKeyStore.getKeyPair()).thenReturn(Pair(mock(), FakeECPublicKey()))
+//
+//        WebAuthProvider.useDPoP()
+//            .login(account)
+//            .start(activity, callback)
+//
+//        verify(activity).startActivity(intentCaptor.capture())
+//        val uri = intentCaptor.firstValue.getParcelableExtra<Uri>(AuthenticationActivity.EXTRA_AUTHORIZE_URI)
+//        assertThat(uri, `is`(notNullValue()))
+//        assertThat(uri, UriMatchers.hasParamWithValue("dpop_jkt", "KQ-r0YQMCm0yVnGippcsZK4zO7oGIjOkNRbvILjjBAo"))
+//    }
+
+    @Test
+    public fun shouldResetDPoPStateWhenManagerInstanceIsReset() {
+        WebAuthProvider.useDPoP()
+
+        // Verify DPoP is enabled
+        val provider1 = WebAuthProvider.useDPoP()
+        assertThat(provider1, `is`(notNullValue()))
```

Comment:
The assertions in this test are just checking that the `WebAuthProvider` instance returned by `useDPoP()` is not null. Is this intended?